app/views/site_page/map.html.erb

```diff
@@ -9,47 +9,40 @@
 
 <% content_for :external_js do %>
   <script id="library-load" src="//wri-sites.s3.amazonaws.com/gfw-mapbuilder.org/library.gfw-mapbuilder.org/<%= @site_page.content['version'] || '1.1.15' %>.js"></script>
-  <script>
-        var mapDatabaseConfig = JSON.parse('<%= @map_content %>')
+  <script type="text/javascript">  
 
+  function ready(fn) {
+    if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading"){
+      fn();
+    } else {
+      document.addEventListener('DOMContentLoaded', fn);
+    }
+  }
+
+  ready(function () {
+    console.log('GON', window.gon);
```

Comment:
Are we going to write this to the console?

Reply:
ofcourse not :) WIP